spec/api.json

```diff
@@ -413,10 +413,57 @@
         },
         "type": "object"
       },
+      "Provider": {
```

Comment:
Should this end up in the API spec?

Reply:
Yes, I think so, because it needs to be part of the SDK.